source/documentation/managing_apps/ssh.erb

```diff
@@ -123,116 +117,92 @@ For example, you can connect directly to the PostgreSQL service bound to an appl
         ...
     ```
 
-    You will need to know:
-    + the remote host, displayed as `"host":`
-    + the remote port, displayed as `"port"`.
-    + the PostgreSQL username, displayed as `"username":`
-    + the PostgreSQL password, displayed as `password:`
-    + the name of the database, displayed as `name:`
+    Make a note of the:
+    - remote `"host":`
+    - remote `"port"`
+    - PostgreSQL `"username":`
+    - PostgreSQL `"password:`
+    - database `"name":`
 
- 2. Create a SSH tunnel using the local port 6666:
+1. Create a local SSH tunnel between local port 6666 and your app's remote host and port:
```

Comment:
This is actually between your local port 6666 and the backing service's host and port. It is tunnelled through the app.